### DISCUSSION TITLE
[PSD11] Exercise 3.4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#174] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210061916-2042a6da-218d-458a-b18c-09f914483516.png)

**Comment by Elia-Belli** [comment]
![d](https://user-images.githubusercontent.com/107944476/213870285-1b588718-bb75-4ccd-82e2-ae6c1f7f5fea.png)

Un D-Latch campiona il segnale $D$ per tutto il periodo in cui $CLK = 1$ e poi ricorda il valore

**Comment by fraxb04** [comment]
@Elia-Belli 
corretto, but stessa cosa del post precedente
